internal/services/bot/bot_channels_registration_resource.go

```diff
@@ -246,20 +221,10 @@ func resourceBotChannelsRegistrationCreate(d *pluginsdk.ResourceData, meta inter
 
 	// d.GetOk cannot identify whether user sets the property that is bool type and `public_network_access_enabled` is set as `false`. So it has to identify it using `d.GetRawConfig()`
 	publicNetworkAccessEnabled := d.GetRawConfig().AsValueMap()["public_network_access_enabled"]
-	if !features.FourPointOhBeta() {
-		// d.GetOk cannot identify whether user sets the property that is bool type and `isolated_network_enabled` is set as `false`. So it has to identify it using `d.GetRawConfig()`
-		isolatedNetworkEnabled := d.GetRawConfig().AsValueMap()["isolated_network_enabled"]
-		if !isolatedNetworkEnabled.IsNull() || !publicNetworkAccessEnabled.IsNull() {
-			return resourceBotChannelsRegistrationUpdate(d, meta)
-		} else {
-			return resourceBotChannelsRegistrationRead(d, meta)
-		}
+	if !publicNetworkAccessEnabled.IsNull() {
+		return resourceBotChannelsRegistrationUpdate(d, meta)
```

Comment:
Why are we calling the update function here?

Reply:
public_network_access_enabled cannot be set successfully in the create operation. It needs to be updated in the update operation. It's service API limitation.